Makefile

```diff
@@ -100,9 +100,9 @@ help-install:
 	@echo "  - rocq-core: Rocq's prelude and corelib"
 	@echo "  - coqide-server: XML protocol language server"
 	@echo "  - rocqide: RocqIDE gtk application"
-	@echo "  - rocq: meta package depending on rocq-runtime rocq-core rocq-stdlib"
+	@echo "  - rocq-prover: meta package depending on rocq-runtime rocq-core rocq-stdlib"
```

Comment:
actually rocq-prover (and coq) isn't this repo anymore so this mention should just be removed I think